case/src/main/kotlin/com/ritense/case/configuration/CaseAutoConfiguration.kt

```diff
@@ -93,6 +96,14 @@ class CaseAutoConfiguration {
         return CaseInstanceResource(service)
     }
 
+    @Bean
+    @ConditionalOnMissingBean(name = ["taskListResource"]) // because integration tests fail to initialise in portaaltaak
```

Comment:
In the future we want to use the `@ConditionalOnMissingBean(MyService::class)` on all beans. So not based on `name` but on `class`. Because in the future:
- All services in Valtimo will get an interface
- Interfaces can be overriden thanks to the `@ConditionalOnMissingBean(MyService::class)`